docs/tutorials/contracts/fts/simple-fts.md

```diff
@@ -0,0 +1,155 @@
+
+## Prerequisites
+
+To complete this tutorial successfully, you'll need:
+
+- [A NEAR account](#wallet)
+- [NEAR command-line interface](/docs/develop/contracts/rust/intro#installing-the-near-cli) (`near-cli`)
+
+### Wallet
+
+To store your fungible tokens you'll need a [NEAR Wallet](https://wallet.testnet.near.org/).
+If you don't have one yet, you can create one easily by following [these instructions](/docs/develop/basics/create-account):
+
+> **Tip:** If you already have a NEAR `testnet` account, you can [skip these steps](#installing-the-near-cli).
+
+#### 1. Reserve an Account ID
+
+- Navigate to https://wallet.testnet.near.org and click on "Create Account".
+- Next, enter your desired account name.
+
+#### 2. Secure your account
+
+- Choose your account recovery method. For simplicity, in this tutorial you can select
+  "E-mail Account Recovery", although "Recovery Phrase" and [Ledger](https://www.ledger.com/)
+  are the most secure methods.
+
+#### 3. E-mail / Phone Number Account Recovery
+
+- Enter the account activation code that you received.
+
+#### 4. Success!
+
+- You just created a `testnet` account and received 200 Ⓝ! Upon recovery method confirmation
+  you should be directed to your account dashboard.
+
+Once you have your Wallet account, you can click on the [Balances Tab](https://wallet.testnet.near.org/?tab=balances) where all your Fungible Tokens will be listed:
+
+![Wallet](/docs/assets/fts/empty-wallet-ft-tab.png)
+
+### Installing the `near-cli`
+
+The following instructions are taken from the `near-cli` [installation
+guide](https://docs.near.org/docs/tools/near-cli#setup). If you already have the command line
+interface, you can [skip these steps](#using-the-ft-contract).
+
+> **Note:** Make sure you have a current version of `npm` and `NodeJS` installed.
+
+#### Linux and macOS
+
+1. Install `npm` and `node` using a package manager such as `nvm`. Sometimes there are issues
+   using Ledger due to how macOS handles node packages related to USB devices.
+   [[click here]](https://nodejs.org/en/download/package-manager/)
+2. Ensure you have installed Node version 12 or above.
+3. Install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
+
+#### Windows
+
+> **Note:** For Windows users, we recommend using Windows Subsystem for Linux (`WSL`).
+
+1. Install `WSL` [[click here]](https://docs.microsoft.com/en-us/windows/wsl/install-manual#downloading-distros)
+2. Install ` Node.js` [[click here]](https://nodejs.org/en/download/package-manager/)
+3. Change `npm` default directory [[click here]](https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally#manually-change-npms-default-directory)
+   - This is to avoid any permission issues with `WSL`
+4. Open `WSL` and install `near-cli` globally by running:
+
+```bash
+npm install -g near-cli
+```
```

Comment:
Same here... we can remove this section and link to the CLI installation docs

Reply:
Fixed